### PR TITLE
Fix timeout on workspace start on Windows

### PR DESCRIPTION
**Pull request by jtary**
Add a check to workspace-start to change the server host to localhost
if bound to 0.0.0.0.

connect to strongloop-internal/scrum-arc#23

### CODE REVIEW

**Comment by jtary** [comment]
@bajtos PTAL

File: common/models/workspace.js [hunk 1]
```diff
@@ -350,13 +350,19 @@ module.exports = function(Workspace) {
...
         var waitOpts = {
           host: host,
           port: port || 3000, // 3000 is the default port provided by loopback
           timeoutInMs: 30000  // 30 seconds
         };
 
+        // Windows will fail to connect if the host is 0.0.0.0, so redirect to
+        // localhost to prevent the startup detection from timing out.
+        if (waitOpts.host === '0.0.0.0') {
+          waitOpts.host = 'localhost';
```
Comment (bajtos) [inline common/models/workspace.js]: FWIW, here is the LoopBack code dealing with this problem:

https://github.com/strongloop/loopback/blob/bfd5059d119d7a7131a04c9873cdc61f04aa1165/lib/application.js#L519-L547

**Comment by bajtos** [comment]
LGTM, thank you for the fix!

**Comment by bajtos** [comment]
Released in `loopback-workspace@3.18.4`.

**Pull request merged**